tsl/test/expected/tsl_tables.out

```diff
@@ -814,73 +814,152 @@ select add_reorder_policy('test_table', 'test_table_time_idx') as job_id \gset
+   1014 | @ 123 years       | @ 7 mins 36 secs |          20 | @ 33 hours   | -infinity
 (1 row)
 
 select * from alter_job_schedule(:job_id, max_retries => NULL);
- job_id | schedule_interval |   max_runtime    | max_retries | retry_period 
---------+-------------------+------------------+-------------+--------------
-   1014 | @ 123 years       | @ 7 mins 36 secs |          20 | @ 33 hours
+ job_id | schedule_interval |   max_runtime    | max_retries | retry_period | next_start 
+--------+-------------------+------------------+-------------+--------------+------------
+   1014 | @ 123 years       | @ 7 mins 36 secs |          20 | @ 33 hours   | -infinity
 (1 row)
 
+--change schedule_interval when bgw_job_stat does not exist
+select * from alter_job_schedule(:job_id, schedule_interval=>'1 min');
+ job_id | schedule_interval |   max_runtime    | max_retries | retry_period | next_start 
+--------+-------------------+------------------+-------------+--------------+------------
+   1014 | @ 1 min           | @ 7 mins 36 secs |          20 | @ 33 hours   | -infinity
+(1 row)
+
+select count(*) = 0 from _timescaledb_internal.bgw_job_stat where job_id = :job_id;
+ ?column? 
+----------
+ t
+(1 row)
+
+--set next_start when bgw_job_stat does not exist
+select * from alter_job_schedule(:job_id, next_start=>'2001-01-01 01:01:01');
+ job_id | schedule_interval |   max_runtime    | max_retries | retry_period |          next_start          
+--------+-------------------+------------------+-------------+--------------+------------------------------
+   1014 | @ 1 min           | @ 7 mins 36 secs |          20 | @ 33 hours   | Mon Jan 01 01:01:01 2001 PST
+(1 row)
+
+--change schedule_interval when no last_finish set
+select * from alter_job_schedule(:job_id, schedule_interval=>'10 min');
+ job_id | schedule_interval |   max_runtime    | max_retries | retry_period | next_start 
+--------+-------------------+------------------+-------------+--------------+------------
+   1014 | @ 10 mins         | @ 7 mins 36 secs |          20 | @ 33 hours   | -infinity
+(1 row)
+
+--next_start overrides any schedule_interval changes
+select * from alter_job_schedule(:job_id, schedule_interval=>'20 min', next_start=>'2002-01-01 01:01:01');
+ job_id | schedule_interval |   max_runtime    | max_retries | retry_period |          next_start          
+--------+-------------------+------------------+-------------+--------------+------------------------------
+   1014 | @ 20 mins         | @ 7 mins 36 secs |          20 | @ 33 hours   | Tue Jan 01 01:01:01 2002 PST
+(1 row)
+
+--set the last_finish manually
+\c :TEST_DBNAME :ROLE_SUPERUSER
+UPDATE _timescaledb_internal.bgw_job_stat SET last_finish = '2003-01-01:01:01:01' WHERE job_id = :job_id;
+\c :TEST_DBNAME :ROLE_DEFAULT_PERM_USER
+--not changing the interval doesn't change the next_start
+select * from alter_job_schedule(:job_id, schedule_interval=>'20 min');
+WARNING:  Timescale License expired
```

Comment:
we probably shouldn't check license expiration on calls of this function if we've taken it out of enterprise

Reply:
but not sure if that's true/if we do it elsewhere...